gc/base/standard/Scavenger.hpp

```diff
@@ -263,7 +263,18 @@ class MM_Scavenger : public MM_Collector
 	 * @return Whether or not objectPtr should be remembered.
 	 */
 	MMINLINE bool scavengeObjectSlots(MM_EnvironmentStandard *env, MM_CopyScanCacheStandard *scanCache, omrobjectptr_t objectPtr, uintptr_t flags, omrobjectptr_t *rememberedSetSlot);
-	MMINLINE MM_CopyScanCacheStandard *incrementalScavengeObjectSlots(MM_EnvironmentStandard *env, omrobjectptr_t objectPtr, MM_CopyScanCacheStandard* scanCache);
+	MMINLINE MM_CopyScanCacheStandard *incrementalScavengeObjectSlots(MM_EnvironmentStandard *env, omrobjectptr_t objectPtr, MM_CopyScanCacheStandard* scanCache);	
+	
+	MMINLINE void
+	deepScan(MM_EnvironmentStandard *env, omrobjectptr_t objectPtr, uintptr_t gcPriorityScanField1, uintptr_t gcPriorityScanField2){
+	
+		if(((uintptr_t)objectPtr & 0xF0) == 0){
```

Comment:
put a comment why is this done

Reply:
Most of configurations will have 8 byte alignment, for which 0x78 will give you better randomization effect. 
For 16 byte alignment, 0xF0 would be better, but I'd rather optimize 8 byte aligment scenario.

We could be fancy and use a variable mask dependent on alignment (and even some other factors), but I'm fine with a const mask.